src/pages/teamMembers.js

```diff
@@ -0,0 +1,245 @@
+import React, { Fragment } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+import GridList from '@material-ui/core/GridList';
+import GridListTile from '@material-ui/core/GridListTile';
+import GridListTileBar from '@material-ui/core/GridListTileBar';
+import Modal from '@material-ui/core/Modal';
+import Backdrop from '@material-ui/core/Backdrop';
+import Zoom from '@material-ui/core/Zoom';
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
move this to separate file and import that here and then call the makeStyles hook